microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/integration/config/CustomMetricCustomConfigSourceProviderTest.java

```diff
@@ -68,22 +67,26 @@ void setConfigProperties(int increment) throws IOException {
 
         @Override
         public void setup(ManagementClient managementClient, String s) throws Exception {
-            OnlineManagementClient client = ManagementClientProvider.onlineStandalone();
-            client.execute(String.format("/system-property=%s:add(value=%s)", CustomConfigSource.FILEPATH_PROPERTY,
-                    SetupTask.class.getResource(PROPERTY_FILENAME).getFile()));
-            ModuleUtil.setupModule(client, new File(SetupTask.class.getResource("configSourceProviderModule.xml").toURI()),
-                    TEST_MODULE_NAME, "config-source-provider", CustomConfigSource.class, CustomConfigSourceProvider.class);
-            client.execute(String.format(
-                    "/subsystem=microprofile-config-smallrye/config-source-provider=cs-from-provider:add(class={module=%s, name=%s})",
-                    TEST_MODULE_NAME, CustomConfigSourceProvider.class.getName()));
+            try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+                client.execute(String.format("/system-property=%s:add(value=%s)", CustomConfigSource.FILEPATH_PROPERTY,
+                        SetupTask.class.getResource(PROPERTY_FILENAME).getPath()));
+                ModuleUtil.add(TEST_MODULE_NAME)
+                        .moduleXMLPath(SetupTask.class.getResource("configSourceProviderModule.xml").getPath())
+                        .resource("config-source-provider", CustomConfigSource.class, CustomConfigSourceProvider.class)
+                        .executeOn(client);
+                client.execute(String.format(
+                        "/subsystem=microprofile-config-smallrye/config-source-provider=cs-from-provider:add(class={module=%s, name=%s})",
+                        TEST_MODULE_NAME, CustomConfigSourceProvider.class.getName()));
+            }
         }
 
         @Override
         public void tearDown(ManagementClient managementClient, String s) throws Exception {
-            OnlineManagementClient client = ManagementClientProvider.onlineStandalone();
-            client.execute(String.format("/system-property=%s:remove", CustomConfigSource.FILEPATH_PROPERTY));
-            client.execute("/subsystem=microprofile-config-smallrye/config-source-provider=cs-from-provider:remove");
-            ManagementClientProvider.onlineStandalone().apply(new RemoveModule(TEST_MODULE_NAME));
+            try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+                client.execute(String.format("/system-property=%s:remove", CustomConfigSource.FILEPATH_PROPERTY));
```

Comment:
I would vote not to use String.format() in CLI command as it's hard to read.

Reply:
This is not valid. Commit should not be here.